src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/client/PrisonApiClientTest.kt

```diff
@@ -113,29 +113,15 @@ class PrisonApiClientTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `retrieve agency, including inactive`() {
-    val request = HttpRequest.request().withPath("/api/agencies/MDI%3FactiveOnly=false")
-
-    prisonApi.`when`(request).respond(
-      HttpResponse.response().withContentType(MediaType.APPLICATION_JSON)
-        .withBody(agencyResponse("MDI")),
-    )
-
-    val agency = prisonApiClient.retrieveAgency("MDI", false).block()
-
-    assertThat(agency?.agencyId, equalTo("MDI"))
-  }
-
-  @Test
-  fun `retrieve agency, active only`() {
-    val request = HttpRequest.request().withPath("/api/agencies/MDI%3FactiveOnly=true")
+  fun `retrieve agency`() {
+    val request = HttpRequest.request().withPath("/api/agencies/MDI")
```

Comment:
Small point: could you add `.withQueryStringParameter("activeOnly", false))` to this to ensure we are always requesting the inactive ones too, please?